src/mscorlib/src/System/Reflection/ExceptionHandlingClause.cs

```diff
@@ -0,0 +1,93 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Globalization;
+using System.Diagnostics.Contracts;
+
+namespace System.Reflection
+{
+    public class ExceptionHandlingClause
+    {
+        #region costructor
+        // This class can only be created from inside the EE.
+        protected ExceptionHandlingClause() { }
+        #endregion
+
+        #region Private Data Members
+        private MethodBody m_methodBody;
```

Comment:
Maybe on some other occasion. The fields are accessed from native code so renaming them means adding more diffs than just this one file.